package.json

```diff
@@ -83,15 +83,15 @@
     "grunt-webpack": "3.1.3",
     "jsdom": "9.12.0",
     "jsdom-global": "2.1.1",
-    "load-grunt-config": "1.0.1",
+    "load-grunt-config": "1.0.2",
```

Comment:
do we need to update load-grunt-config and node-sass?

Reply:
npm shows security warning suggesting update

Reply:
npm 6.4.1